images/linux/scripts/helpers/install.sh

```diff
@@ -33,4 +33,13 @@ download_with_retries() {
 
     echo "Could not download $URL"
     return 1
+}
+
+## Use dpkg to figure out if a package has already been installed
+## Example use:
+## if ! IsInstalled packageName; then
+##     echo "packageName is not installed!"
+## fi
+function IsInstalled {
```

Comment:
The function name looks too generic for me.
`IsInstalled` sounds like we check if particular tool is installed on machine but actually this function checks only dpkg.

Reply:
we can rename it to IsPackageInstalled, does it sound better?

Reply:
Yes, I think so, thanks